src/mixxxmainwindow.cpp

```diff
@@ -372,6 +379,11 @@ void MixxxMainWindow::initialize() {
     // Show the menubar after the launch image is replaced by the skin widget,
     // otherwise it would shift the launch image shortly before the skin is visible.
     m_pMenuBar->show();
+#ifndef __APPLE__
+    // If we went fullscreen earlier and the desktop features a global menu
+    // this will signal the menu to move into the window and hide itself
```

Comment:
I don't understand that comment can you rephrase it?

Reply:
I could but I notice it's not required and doesn't belong here. It was required in #11526 at some point for hiding the menubar  after starting in fullscreen mode.

Will remove it.